src/blueprint/operators/grpc.rs

```diff
@@ -166,8 +166,8 @@ pub fn compile_grpc(inputs: CompileGrpc) -> Valid<Expression, String> {
     Valid::from(GrpcMethod::try_from(grpc.method.as_str()))
         .and_then(|method| {
             Valid::from_option(
-                config_module.extensions.get_file_descriptor_set(&method),
-                format!("File descriptor not found for method: {}", grpc.method),
+                config_module.extensions.get_file_descriptor_set(),
```

Comment:
We need to validate if the path to the method exists in the proto file.

Reply:
we have multiple proto files and we'll check that method exists later when executing method resolving

Reply:
Understood, but we are dropping a significant feature here, I don't see why we need to move that check to runtime when we can identify the issue at compile time.

Reply:
The check it later in this file when calling `to_operation` and it's at compile time.

The error looks like this https://github.com/tailcallhq/tailcall/blob/2ddb53663be9b6204187076ce286bb56a0d8df71/tests/snapshots/execution_spec__test-grpc-service.md_errors.snap#L7